sources/packages/web/src/constants/award-constants.ts

```diff
@@ -55,22 +55,22 @@ export const AWARDS: AwardDetail[] = [
   },
   {
     awardType: PartTimeAwardTypes.CSLP,
-    description: "Canada Student Loan for Part-time Studies",
```

Comment:
Re-ordering mentioned in the AC is missing. 

![image](https://github.com/user-attachments/assets/39556799-2f6c-4cb6-b649-598d7d57d811)


![image](https://github.com/user-attachments/assets/4c01dfa6-133e-4fdc-956e-b72d3e37dc69)

Reply:
Updated.